src/components/base/BaseModal.vue

```diff
@@ -1,20 +1,43 @@
 <template>
 	<transition name="modal">
 		<div v-if="active" class="modal-mask">
-			<div class="modal-wrapper" @click.self="handleBackgroundClick">
+			<div class="base-modal-wrapper" @click.self="handleBackgroundClick">
 				<div
 					class="modal-container"
 					:class="{ 'modal-container--large': size === 'large' }"
 				>
-					<slot />
+					<slot>
+						<!-- TODO add styles so footer is always visbile -->
+						<h4 class="modal-header">
```

Comment:
Semanticly I would use something like `h1` or `h2` here and adding a class like `h4` for styling purposes (h4 is a global helper class defined at `src\themes\default\styles\utility\_typography.scss`)
```suggestion
						<h2 class="h4 modal-header">
```